### DISCUSSION TITLE
Fixed type mismatch in remove_from_cache_action callback to ddb_action_context_t

### DISCUSSION THREAD

**Discussion by jeffguy77** [community/community#129529] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When trying to compile, I get the error:
```
$ make gtk3
g++ src/ui.cpp -c `pkg-config --cflags gtkmm-3.0 gtk+-3.0` -std=c++14 -Wall -O2 -fPIC -fvisibility=hidden -flto
g++ src/utils.cpp -c `pkg-config --cflags gtkmm-3.0 gtk+-3.0` -std=c++14 -Wall -O2 -fPIC -fvisibility=hidden -flto
cc -std=c99 -Wall -O2 -D_GNU_SOURCE -fPIC -fvisibility=hidden -flto src/main.c -c `pkg-config --cflags gtk+-3.0`
src/main.c:31:22: error: initialization of ‘int (*)(struct DB_plugin_action_s *, ddb_action_context_t)’ {aka ‘int (*)(struct DB_plugin_action_s *, enum ddb_action_context_e)’} from incompatible pointer type ‘int (*)(DB_plugin_action_t *, int)’ {aka ‘int (*)(struct DB_plugin_action_s *, int)’} [-Wincompatible-pointer-types]
   31 |         .callback2 = remove_from_cache_action,
      |                      ^~~~~~~~~~~~~~~~~~~~~~~~
src/main.c:31:22: note: (near initialization for ‘remove_action.callback2’)
make: *** [Makefile:31: main.o] Error 1
```

So I modified:

src/utils.h line 40
```
int remove_from_cache_action(DB_plugin_action_t *, int ctx);
```
to
```
int remove_from_cache_action(DB_plugin_action_t *, ddb_action_context_t ctx);
```

And

src/utils.cpp line 196
```
int remove_from_cache_action(DB_plugin_action_t *, int ctx) {
```
to
```
int remove_from_cache_action(DB_plugin_action_t *, ddb_action_context_t ctx) {
```

And it compiled successfully. :)

**Comment by ebndev** [comment]
Hi @jeffguy77, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? 

That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
